Formula/d/dpcmd.rb

```diff
@@ -0,0 +1,25 @@
+class Dpcmd < Formula
+  desc "Linux software for DediProg SF100/SF600"
+  homepage "https://github.com/DediProgSW/SF100Linux"
+  url "https://github.com/DediProgSW/SF100Linux/archive/refs/tags/V1.14.20.x.tar.gz"
+  sha256 "d3e710c2a4361b7a82e1fee6189e88a6a6ea149738c9cb95409f0a683e90366e"
+  license "GPL-2.0-only"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  def install
+    system "make"
+    bin.install "dpcmd"
+  end
+
+  test do
+    # Try and read from a device that isn't connected
+    shell_output("#{bin}/dpcmd -rSTDOUT -a0x100 -l0x23", 1)
```

Comment:
Sure, although all it prints is the version string, which changes on each release, so I didn't think it would be useful to check it (since that means the next release would cause this formula to fail). That's why I was only checking the return code to know that it failed, since that shouldn't change between releases

If my understanding isn't correct (e.g., that a new release breaking this test is acceptable/encouraged), I'm happy to change it!

Reply:
Then we can do:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/dpcmd -rSTDOUT -a0x100 -l0x23", 1)
```